CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.0-beta.18 (2017-04-21)
+
+- Allowing `skyux test` to not fail if no `*.spec.ts` files are specified.
```

Comment:
Would it be any clearer to say "Preventing `skyux test` from failing if no..."? I had to read the "Allowing.. not to fail" construction twice to get it.